jdaviz/configs/specviz/plugins/unit_conversion/unit_conversion.py

```diff
@@ -76,6 +81,10 @@ def __init__(self, *args, **kwargs):
         self.flux_unit = UnitSelectPluginComponent(self,
                                                    items='flux_unit_items',
                                                    selected='flux_unit_selected')
```

Comment:
Should we have separate `flux_unit` and `sb_unit` and the `flux_or_sb` switch just changes which is visible (instead of swapping out all the choices).  Or, alternatively, should this be renamed to `flux_or_sb_unit`?  

Either way, we should also think about how this affects specviz where we probably will not allow translating since we won't know the scale factor, but the input spectra could either be in flux or surface brightness 🤔

Reply:
I have added a return statement at the beginning of: https://github.com/spacetelescope/jdaviz/blob/17993097a56eeefe8b50cd2bf979647c1240aff5/jdaviz/configs/specviz/plugins/unit_conversion/unit_conversion.py#L185 It is also disabled through the vue file. You can still 'see' the bool but it doesn't do anything (before if you tried you would get the ValueError thrown that there isn't a scale factor). If there are additional ways to disable between configs I will make sure to include them in this PR. A following comment addresses flux/sb_unit